docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/rs-configuration.md

```diff
@@ -182,16 +182,270 @@ The capacity usage on a pool can be monitored using [exporter metrics](../replic
+spec:
+  node: worker-node-2
+  disks: ["/dev/sdb"]
+  topology:
+    labelled:
+        zone: us-west-1
+ ```
+
+ **Command (Get filtered pools based on labels)**
+ ```text
+kubectl mayastor get pools -n openebs --selector zone=eu-west-1
+ID             GRPC ENDPOINT        STATUS  LABELS
+ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
+pool-on-node-0  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-0  Online  10GiB     0 B        10GiB      0 B
+pool-on-node-2  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-2  Online  10GiB     0 B        10GiB      0 B
+
+kubectl mayastor get pools -n openebs --selector zone=eu-east-1
+ID             GRPC ENDPOINT        STATUS  LABELS
+ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
+pool-on-node-1  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-1  Online  10GiB     0 B        10GiB      0 B
+```
+
+For the case shown above, the replica of the volume will be placed on `pool-on-node-0` and `pool-on-node-3` only as they match the labels specified under `poolAffinityTopologyLabel` in the storage class that is equal to zone=us-west-1.
+
+### "PoolHasTopologyKey"
+
+The parameter 'PoolHasTopologyKey' will allow the placement of replicas on the pool that has label keys same as the keys passed in the storage class.
+
+**Command**
+```text
+cat <<EOF | kubectl create -f -
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: mayastor-1
+parameters:
+  ioTimeout: "30"
+  protocol: nvmf
+  repl: "2"
+  poolHasTopologykey: |
+    zone
+provisioner: io.openebs.csi-mayastor
+volumeBindingMode: Immediate
+```
+
+**Command (Get filtered pools based on labels)**
+ ```text
+kubectl mayastor get pools -n openebs --selector zone=eu-west-1
+ID             GRPC ENDPOINT        STATUS  LABELS
+ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
+pool-on-node-0  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-0  Online  10GiB     0 B        10GiB      0 B
+pool-on-node-2  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-2  Online  10GiB     0 B        10GiB      0 B
+
+kubectl mayastor get pools -n openebs --selector zone=eu-east-1
+ID             GRPC ENDPOINT        STATUS  LABELS
+ID              DISKS                                                     MANAGED  NODE           STATUS  CAPACITY  ALLOCATED  AVAILABLE  COMMITTED
+pool-on-node-1  aio:///dev/sdb?uuid=b7779970-793c-4dfa-b8d7-03d5b50a45b8  true     worker-node-1  Online  10GiB     0 B        10GiB      0 B
+```
+
+In this case, the replica of the volume will be placed on any of the two pools i.e.
```

Comment:
In this case, the replica of the volume will be placed on any of the three pools i.e.
- `pool-on-node-1` and `pool-on-node-2` or
- `pool-on-node-1` and `pool-on-node-3` or
- `pool-on-node-2` and `pool-on-node-3`
as the storage class has `zone` as the value for `poolHasTopoogyKey` that matches with the label key of the pool.

Reply:
"any two of the three pools", right?